codeship-steps.yml

```diff
@@ -0,0 +1,40 @@
+-
+  name: Test
+  type: parallel
+  steps:
+    -
+      service: shipyard
+      name: Jekyll
+      command: ./ci/jekyll
+    -
+      service: shipyard
+      name: RSpec
+      command: rspec
+    -
+      service: shipyard
+      name: Sass
+      command: ./ci/sass_lint
+
+-
+  name: Review
+  type: manual
+  exclude: master
+  steps:
+    -
+      service: shipyard
+      name: Percy
+      command: ./ci/percy
+
+-
+  name: Deploy
+  type: serial
+  tag: master
```

Comment:
@brettbuddin @markphelps When I comment this line out and run `jet steps` I get this:

<img width="392" alt="screenshot 2018-04-24 14 27 23" src="https://user-images.githubusercontent.com/211597/39206600-e5cf22f8-47cb-11e8-80ee-caad9b8c32be.png">

Will this be an issue at all when it goes into production since I'm skipping the `Review` step on master?

Reply:
yeah, so I believe you actually want the deploy step to be manual.  Are you trying to manually approve deployment?

Reply:
I'm not super familiar with the manual step behavior, so you probably want to wait on some commentary from the others.

Reply:
@jondavidjohn I actually don't want the deploy step to be manual. It's already largely manual in that I click a button already to merge the PR into master for deployment.

In this case, I want to approve the review step to run so I can review the work on Percy first and confirm that the changes are what I want before I merge the PR into master. I've already accomplished this flow in Jenkins.

Reply:
@ryanjwilke The `type: manual` step needs to be the last step on a build (for a specific tag). When you remove the `tag: master` from the _Deploy_ step, there is another step (the _Deploy_) run after the manual approval (at least on branches other than `master`), which causes the error message.

What's the exact use case? From the current steps file I'd assume you want to run the manual approval on all branches, but want to deploy only from `master`? Why do you want to remove the `tag: master` from the deploy step? 

If you want to test this locally, you can pass in the tag via the CLI, e.g. `jet steps --tag master`